package.json

```diff
@@ -34,17 +34,19 @@
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike#readme",
   "devDependencies": {
     "@crowdstrike/commitlint": "^7.0.0",
-    "eslint": "^8.0.0",
-    "eslint-config-crowdstrike": "link:",
```

Comment:
Why was this link dropped?

Reply:
Thanks for answering this in the sync, but for posterity:

new release of `eslintv9` compatible https://github.com/CrowdStrike/eslint-config-crowdstrike-node expects a `eslintv8` version of this repo, so pinning to the `eslintv8` version of this repo `@10.1.0`.